### PR TITLE
[Integration][Statuspage, AWS, GCP] Add _target='blank' attribute to spec links

### PR DESCRIPTION
**Pull request by oiadebayo**
# Description

What - Added `_target='blank'` attribute to specification links to ensure they open in a new browser tab instead of the current browser tab.
Why - To enhance user experience by allowing users to keep their current page open while exploring specification links in new tabs. This prevents disruptions and allows for easier reference between the current page and the new tab.
Added _target='blank' attribute to spec links to open a new browser tab instead of the current browser
How - Updated the HTML attributes of the specification links by adding `_target='blank'`.

## Type of change

Please leave one option from the following and delete the rest:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] New Integration (non-breaking change which adds a new integration)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Non-breaking change (fix of existing functionality that will not change current behavior)
- [ ] Documentation (added/updated documentation)

### CODE REVIEW

File: integrations/aws/.port/spec.yaml [hunk 1]
```diff
@@ -15,12 +15,12 @@ configurations:
     required: false
     type: string
     sensitive: true
-    description: AWS User Access Key ID, for more information https://docs.aws.amazon.com/singlesignon/latest/userguide/howtogetcredentials.html.
+    description: The AWS User Access Key ID used for authentication. For more information, visit the <a href=”https://docs.aws.amazon.com/singlesignon/latest/userguide/howtogetcredentials.html” target=”_blank”>AWS documentation</a>
```
Comment (PeyGis) [inline integrations/aws/.port/spec.yaml:18]:
```suggestion
    description: The AWS User Access Key ID used for authentication. For more information, visit the <a href="https://docs.aws.amazon.com/singlesignon/latest/userguide/howtogetcredentials.html" target="_blank">AWS documentation</a>
```

File: integrations/aws/.port/spec.yaml [hunk 2]
```diff
@@ -15,12 +15,12 @@ configurations:
...
   - name: awsSecretAccessKey
     required: false
     type: string
     sensitive: true
-    description: AWS User Secret Access Key, for more information https://docs.aws.amazon.com/singlesignon/latest/userguide/howtogetcredentials.html.
+    description: The AWS User Secret Access Key used for authentication. For more information, see the <a href=”https://docs.aws.amazon.com/singlesignon/latest/userguide/howtogetcredentials.html” target=”_blank”>AWS documentation</a>.
```
Comment (PeyGis) [inline integrations/aws/.port/spec.yaml:23]: Check the string quote. Use "" instead of ””

**Review by PeyGis — CHANGES_REQUESTED** [review]
left some comment

File: integrations/aws/.port/spec.yaml [hunk 3]
```diff
@@ -15,12 +15,12 @@ configurations:
...
     type: string
     sensitive: true
-    description: AWS User Access Key ID, for more information https://docs.aws.amazon.com/singlesignon/latest/userguide/howtogetcredentials.html.
+    description: The AWS User Access Key ID used for authentication. For more information, visit the <a href="https://docs.aws.amazon.com/singlesignon/latest/userguide/howtogetcredentials.html" target="_blank">AWS documentation</a>
   - name: awsSecretAccessKey
     required: false
     type: string
     sensitive: true
-    description: AWS User Secret Access Key, for more information https://docs.aws.amazon.com/singlesignon/latest/userguide/howtogetcredentials.html.
+    description: The AWS User Secret Access Key used for authentication. For more information, see the <a href=”https://docs.aws.amazon.com/singlesignon/latest/userguide/howtogetcredentials.html” target=”_blank”>AWS documentation</a>.
```
Comment (Tankilevitch) [inline integrations/aws/.port/spec.yaml:23]:
```suggestion
    description: The AWS User Secret Access Key used for authentication. For more information, see the <a href="https://docs.aws.amazon.com/singlesignon/latest/userguide/howtogetcredentials.html" target="_blank">AWS documentation</a>.
```

File: integrations/gcp/.port/spec.yaml [hunk 4]
```diff
@@ -9,7 +9,7 @@ configurations:
   - name: encodedADCConfiguration
     type: string
     require: false
-    description: Base64 encoding of your Google Cloud's ADC Credentials configuration file. Go to https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/cloud-providers/gcp/installation for more information
+    description: Base64 encoding of your Google Cloud's ADC Credentials configuration file. For more information, see the <a href=”https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/cloud-providers/gcp/installation” target=”_blank”>Google documentation</a>
```
Comment (Tankilevitch) [inline integrations/gcp/.port/spec.yaml:12]:
```suggestion
    description: Base64 encoding of your Google Cloud's ADC Credentials configuration file. For more information, see the <a href="https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/cloud-providers/gcp/installation" target="_blank">Google documentation</a>
```

File: integrations/statuspage/.port/spec.yaml [hunk 5]
```diff
@@ -14,18 +14,18 @@ configurations:
...
-    description: "The host of the Statuspage.io API. The default host is https://api.statuspage.io"
-    default: "https://api.statuspage.io"
+    description: The host of the Statuspage.io API. The default value is https://api.statuspage.io
+    default: https://api.statuspage.io
   - name: statuspageApiKey
     required: true
     type: string
     sensitive: true
-    description: Statuspage API key. To find your API key, see the <a href="https://support.atlassian.com/statuspage/docs/create-and-manage-api-keys/">Statuspage documentation</a>.
+    description: Statuspage API key. To find your API key, see the <a href="https://support.atlassian.com/statuspage/docs/create-and-manage-api-keys/" target=”_blank”>Statuspage documentation</a>.
```
Comment (Tankilevitch) [inline integrations/statuspage/.port/spec.yaml:27]:
```suggestion
    description: Statuspage API key. To find your API key, see the <a href="https://support.atlassian.com/statuspage/docs/create-and-manage-api-keys/" target="_blank">Statuspage documentation</a>.
```

File: integrations/statuspage/pyproject.toml [hunk 6]
```diff
@@ -1,12 +1,13 @@
...
 name = "statuspage"
-version = "0.1.10"
+version = "0.1.11"
 description = "Connect Statuspage to Ocean and automatically ingest incidents, updates, and impacted components for comprehensive monitoring"
 authors = ["Albert Luganga <albertluganga@getport.io>"]
 
 [tool.poetry.dependencies]
 python = "^3.11"
-port_ocean = {version = "^0.9.7", extras = ["cli"]}
+port_ocean = {extras = ["cli"], version = "^0.9.7"}
```
Comment (Tankilevitch) [inline integrations/statuspage/pyproject.toml:9]:
```suggestion
port_ocean = {version = "^0.9.7", extras = ["cli"]}
```

File: integrations/statuspage/pyproject.toml [hunk 7]
```diff
@@ -1,13 +1,14 @@
...
-version = "0.1.10"
+version = "0.1.11"
 description = "Connect Statuspage to Ocean and automatically ingest incidents, updates, and impacted components for comprehensive monitoring"
 authors = ["Albert Luganga <albertluganga@getport.io>"]
 
 [tool.poetry.dependencies]
 python = "^3.11"
 port_ocean = {version = "^0.9.7", extras = ["cli"]}
 
+
```
Comment (Tankilevitch) [inline integrations/statuspage/pyproject.toml:11]:
```suggestion
```

**Review by Tankilevitch — APPROVED** [review]

**Pull request merged**